dspace-server-webapp/src/test/resources/test-config.properties

```diff
@@ -14,3 +14,7 @@ test.bitstream = ./target/testing/dspace/assetstore/ConstitutionofIreland.pdf
 
 #Path for a test Taskfile for the curate script
 test.curateTaskFile = ./target/testing/dspace/assetstore/curate.txt
+
+test.crossRef = ./target/testing/dspace/assetstore/crossRef-test.json
+test.vufind = ./target/testing/dspace/assetstore/vuFind-generic.json
+test.vufind-by-id = ./target/testing/dspace/assetstore/vuFind-by-id.json
```

Comment:
Again, if these files are moved to `dspace-server-webapp`, then these configs can be removed